diagnostic.md

```diff
@@ -16,13 +16,13 @@ Graces-MacBook-Pro:try grace$
 
 What is the full path to the current directory?
 
-_Replace this text with your answer._
+wdi grace try
```

Comment:
The current directory's full path is /Users/grace/wdi/try.  While pwd showed "/Users/grace/wdi", we created a directory called "try" afterward and then entered it with "cd try".